core/services/gasupdater/gas_updater_test.go

```diff
@@ -482,7 +483,7 @@ func TestGasUpdater_Recalculate(t *testing.T) {
 		config.On("GasUpdaterTransactionPercentile").Return(uint16(50))
 		config.On("ChainID").Return(big.NewInt(0))
 
-		guIface := gasupdater.NewGasUpdater(ethClient, config)
+		guIface := gasupdater.NewGasUpdater(ethClient, config, headtracker.NewHeadBroadcaster())
```

Comment:
Does it work if we use the null pattern to pass in a null head broadcaster?

Or perhaps a mock?

Reply:
removed them now, as it's now being subscribed in application.go